Dockerfile

```diff
@@ -9,8 +9,37 @@ RUN apk add --no-cache npm \
   && npm i /src \
   && npm run build
 
-FROM nginx:latest
+################################################
 
-COPY --from=build  /src/build /usr/share/nginx/html
+FROM alpine:3 as lighttpd-builder
+ARG LIGHTTPD_VERSION="1.4.76"
```

Comment:
why not use greatest version (which seems to be .77 as of today?)

Reply:
also: if building it using alpine, why not instead just install it from there (https://wiki.alpinelinux.org/wiki/Lighttpd)?

or: use https://hub.docker.com/r/jitesoft/lighttpd 🤔

Reply:
package provided by alpine is not statically linked against external dependencies, see [build description](https://git.lighttpd.net/lighttpd/lighttpd1.4/src/commit/4bdd6363e26c6f5a6de1df82d17e3fa99416c282/INSTALL#L161)

this makes it very hard to consume in a scratch image

Reply:
will check w.r.t. latest version
x.x.76 was latest available package on alpine package (my first approach was based on that package), I did not check afterwards whether a greater version is available if building from scratch 👍